eng/Versions.props

```diff
@@ -36,8 +36,8 @@
   </PropertyGroup>
   <!-- Toolset Dependencies -->
   <PropertyGroup>
-    <DotNetCliVersion>3.1.100</DotNetCliVersion>
-    <MicrosoftNetCompilersToolsetVersion>3.3.1-beta3-final</MicrosoftNetCompilersToolsetVersion>
+    <DotNetCliVersion>5.0.100</DotNetCliVersion>
```

Comment:
Should probably be .101 (or .102) now.

Reply:
Where can I go to verify this? I did a brief search in the sdk and arcade repos

Reply:
https://dotnet.microsoft.com/download/dotnet/5.0 says 102 is current latest

Reply:
🤦‍♂️ 👍